kolibri/plugins/user/assets/src/views/profile-page/index.vue

```diff
@@ -99,11 +109,40 @@
         }
         return '';
       },
+      canEditUsername() {
```

Comment:
This block is duplicated in this method and the nexxt:
```
        if (this.isSuperuser) {
 +          return false;
 +        } else if (this.isAdmin) {
 +          return true;
```

Doesn't mean it needs to be de-duplicated, but may hint at a more basic concept behind the idea that 
superusers cannot edit their name/username; admins can (and for learners it depends).